microsoft/testsuites/network/netinterface.py

```diff
@@ -123,7 +123,7 @@ def verify_network_interface_reload_via_ip_link(
         # Ensure netvsc module is loaded
         network_interface_feature.reload_module()
 
-        node_nic_info = Nics(node)
+        node_nic_info = create_nics(node)
```

Comment:
replace to node.nics.

Reply:
I will update this in my next PR dealing with network testcases. This would give me a chance to verify the change properly.

Reply:
@squirrelsc Is this okay?

Reply:
Can all `create_nics` change to node.nics?

Reply:
If any of the test case changes the nic information like ip or disable mlx driver, we would need to call `create_nic` again since the previous information is outdated

Reply:
In this case, it should call nics.reload to make it meaningful.

Reply:
Discussed offline. I will address this change soon in a separate PR